readme.md

```diff
@@ -350,6 +350,7 @@ Thanks for contributing! 🦋🙌
 - [](# "linkify-notification-repository-header") [Linkifies the header of each notification group (when grouped by repository).](https://user-images.githubusercontent.com/1402241/80849887-81531c00-8c19-11ea-8777-7294ce318630.png)
 - [](# "stop-redirecting-in-notification-bar") [Stops redirecting to notification inbox from notification bar actions while holding <kbd>Alt</kbd>.](https://user-images.githubusercontent.com/202916/80318782-c38cef80-880c-11ea-9226-72c585f42a51.png)
 - [](# "last-notification-page-button") [Adds a link to the last page of notifications.](https://user-images.githubusercontent.com/16872793/199828181-3ff2cef3-8740-4efa-8122-8f2f222bd657.png)
+- [](# "pr-notification-link") [Points PR notifications to the conversation tabs instead of the commits page, which may be a 404.]()
```

Comment:
The feature checker still isn't strict enough. I almost merged this 🥲

Reply:
Can anyone take a small screenshot of this whole page when that happens? With the light theme, something like this, I'll clean it up:

<img width="972" alt="Screenshot" src="https://github.com/refined-github/refined-github/assets/1402241/c75802d3-c7c2-4f0e-83b4-e761613c6c6b">

None of my notifications point here.

Reply:
Eh, direct link: 😅

https://github.com/refined-github/refined-github/pull/7104/files/super-easy